po/it.po

```diff
@@ -2470,6 +2461,196 @@ msgstr ""
 "prevedibili come C e C++ (nessun Garbage Collector) così come l'accesso "
 "all'hardware di basso livello (quando serve)"
 
+#: src/why-rust/an-example-in-c.md:4
+msgid "Let's consider the following \"minimum wrong example\" program in C:"
+msgstr ""
+
+#: src/why-rust/an-example-in-c.md:6
+msgid ""
+"```c,editable\n"
+"#include <stdio.h>\n"
+"#include <stdlib.h>\n"
+"#include <sys/stat.h>\n"
+"\n"
+"int main(int argc, char* argv[]) {\n"
+"\tchar *buf, *filename;\n"
+"\tFILE *fp;\n"
+"\tsize_t bytes, len;\n"
+"\tstruct stat st;\n"
+"\n"
+"\tswitch (argc) {\n"
+"\t\tcase 1:\n"
+"\t\t\tprintf(\"Too few arguments!\\n\");\n"
+"\t\t\treturn 1;\n"
+"\n"
+"\t\tcase 2:\n"
+"\t\t\tfilename = argv[argc];\n"
+"\t\t\tstat(filename, &st);\n"
+"\t\t\tlen = st.st_size;\n"
+"\t\t\t\n"
+"\t\t\tbuf = (char*)malloc(len);\n"
+"\t\t\tif (!buf)\n"
+"\t\t\t\tprintf(\"malloc failed!\\n\", len);\n"
+"\t\t\t\treturn 1;\n"
+"\n"
+"\t\t\tfp = fopen(filename, \"rb\");\n"
+"\t\t\tbytes = fread(buf, 1, len, fp);\n"
+"\t\t\tif (bytes = st.st_size)\n"
+"\t\t\t\tprintf(\"%s\", buf);\n"
+"\t\t\telse\n"
+"\t\t\t\tprintf(\"fread failed!\\n\");\n"
+"\n"
+"\t\tcase 3:\n"
+"\t\t\tprintf(\"Too many arguments!\\n\");\n"
+"\t\t\treturn 1;\n"
+"\t}\n"
+"\n"
+"\treturn 0;\n"
+"}\n"
+"```"
+msgstr ""
+
+#: src/why-rust/an-example-in-c.md:48
+msgid "How many bugs do you spot?"
+msgstr ""
```

Comment:
No translation for this and the follwoing?

Reply:
Nuova entry, non ancora tradotta